src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -1,86 +1,72 @@
+import com.volunteer.enterprise.service.ICompanyService;
 import com.volunteer.enterprise.service.IVolunteerService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.ui.Model;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
 
 import java.util.List;
 
 @Controller
 public class VolunteerAppController {
+    @Autowired
     IVolunteerService volunteerService;
 
+    @Autowired
+    ICompanyService companyService;
+
     /**
-     *RequestMapping for root (/) endpoint
-     * Create a Voluntier volunteer object then display start
+     * GetMapping for root(/) endpoint
+     * Create a Voluntier volunteer object and then display start
+     *
      * @return Voluntier index page
      */
-
-    @RequestMapping("/")
+    @GetMapping("/")
     public String index(Model model){
-
-    Volunteer volunteer = new Volunteer();
-    volunteer.setVolunteerID(1);
-    volunteer.setVolunteerUser("Iron Man");
-    volunteer.setVolunteerPass("1234");
-    model.addAttribute(volunteer);
+        Volunteer volunteer = new Volunteer();
+        volunteer.setVolunteerID(1);
+        volunteer.setVolunteerEmail("tony@starkindustries.com");
+        volunteer.setVolunteerUser("Iron Man");
+        volunteer.setVolunteerPass("1234");
 
         // Add newly created favorite to model so it can be displayed
         model.addAttribute(volunteer);
         return "Index";
-<<<<<<< HEAD
-
     }
 
-
-
-
     /**
-     *RequestMapping for root (/) endpoint
-     * Create a Company company object then display company form page
+     * GetMapping for /company endpoint
+     * Create a company object and then display company form page
+     *
      * @return company form page
      */
-
-    @RequestMapping("/company")
+    @GetMapping("/company")
     public String company(Model model){
-
         Company company = new Company();
         company.setCompanyEmail("vertex@gmail.com");
         company.setCompanyPass("Man");
         company.setCompanyID(1);
-        model.addAttribute(company);
+        company.setCompanyUser("Bat");
 
         // Add newly created favorite to model so it can be displayed
         model.addAttribute(company);
         return "company";
-=======
->>>>>>> fceb4580fa93faea0c07e0a97c52f04f1117c6f3
-
     }
 
-
-
-
-
-
-
     /**
-     * RequestMapping for /saveVolunteer endpoint
-     * Save a nww volunteer with details provided via HTTP query string
+     * GetMapping for /saveVolunteer endpoint
+     * Save a new volunteer with details provided via HTTP query string
+     *
      * @param volunteer provided through HTTP query
      * @return Volunteer Index page displaying newly saved volunteer
      */
-
-
-
-    @RequestMapping("/saveVolunteer")
+    @GetMapping("/saveVolunteer")
```

Comment:
Changed `@RequestMapping` to the more specific `@GetMapping` where applicable. [Reference](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/bind/annotation/GetMapping.html)